### PR TITLE
To add a report in Microsoft Teams with Power BI Tab, user must have …

### PR DESCRIPTION
**Pull request by tusharvaja**
…at-least viewer permission to workspace hosting the report

As per my discussion with Lukasz Pawlowski, this needs to corrected.

@LukaszPawlowski-MS 

As per this doc related to TEAMS V2 tabs,  to add a report via Power BI tab, “User must have permission to view the report”.
 
In this case, User1 has shared a report with User2.  User2 can see report in his “Shared with Me” section in powerbi.com and can access report from there (so he can view).  User2 does not have access to report workspace and cannot see report workspace in app.powerbi.com.
But when User2 try to search for that report while adding via TEAMS tab , he does not find it. is this expected? if it is expected, then our public doc needs to be corrected as adding in TEAMs tab does requires permission on workspace , only view permission does not work.

### CODE REVIEW

**Comment by PRMerger20** [comment]
@tusharvaja : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: powerbi-docs/collaborate-share/service-embed-report-microsoft-teams.md [hunk 1]
```diff
@@ -23,7 +23,8 @@ For the **Power BI tab for Microsoft Teams** to work, ensure:
 - Your users have a Power BI Pro license, or the report is contained in a [Power BI Premium capacity (EM or P SKU)](../admin/service-premium-what-is.md) with a Power BI license.
 - Microsoft Teams has the Power BI tab.
 - Users have signed in to the Power BI service to activate their Power BI license to consume the report.
-- Users must have permission to view the report.
+- To add a report in Microsoft Teams with Power BI Tab, user must have at-least viewer permission to workspace hosting the report.
```
Comment (maggiesMSFT) [inline powerbi-docs/collaborate-share/service-embed-report-microsoft-teams.md:26]:
```suggestion
- To add a report in Microsoft Teams with the Power BI tab, you must have at least a Viewer role in the workspace hosting the report. See [Roles in the new workspaces](service-new-workspaces.md#roles-in-the-new-workspaces) for information about the different roles.
```

File: powerbi-docs/collaborate-share/service-embed-report-microsoft-teams.md [hunk 2]
```diff
@@ -23,7 +23,8 @@ For the **Power BI tab for Microsoft Teams** to work, ensure:
 - Your users have a Power BI Pro license, or the report is contained in a [Power BI Premium capacity (EM or P SKU)](../admin/service-premium-what-is.md) with a Power BI license.
 - Microsoft Teams has the Power BI tab.
 - Users have signed in to the Power BI service to activate their Power BI license to consume the report.
-- Users must have permission to view the report.
+- To add a report in Microsoft Teams with Power BI Tab, user must have at-least viewer permission to workspace hosting the report. 
+- To see the report in Power BI Tab in Micrrosoft Teams, user must have permission to view the report.
```
Comment (maggiesMSFT) [inline powerbi-docs/collaborate-share/service-embed-report-microsoft-teams.md:27]:
```suggestion
- To see the report in the Power BI tab in Microsoft Teams, users must have permission to view the report.
```

**Comment by PRMerger16** [comment]
@maggiesMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@maggiesMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@maggiesMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by maggiesMSFT** [comment]
#sign-off

**Pull request merged**